mp/src/game/client/momentum/ui/controls/UserComponent.cpp

```diff
@@ -60,11 +60,8 @@ void UserComponent::SetUser(uint64 uID)
     CHECK_STEAM_API(SteamUser());
 
     const CSteamID userID(uID);
-
     m_pUserImage->SetPlayer(userID, k_EAvatarSize184x184);
 
-    m_pUserName->SetText(SteamFriends()->GetFriendPersonaName(userID));
```

Comment:
Should not be removed, as this component may be used for other users in the future.